armory/datasets/preprocessing.py

```diff
@@ -58,6 +58,47 @@ def carla_over_obj_det_dev(element, modality="rgb"):
     )
 
 
+def carla_video_tracking_preprocess(x, max_frames=None):
```

Comment:
input preprocessing also needs to be scaling from [0, 255] to [0, 1]

Reply:
Addressed with e466546acf85008b8edf5815816c4822ee733764